ckan-dev/2.7/Dockerfile

```diff
@@ -8,12 +8,17 @@ ENV GIT_BRANCH=ckan-2.7.6
 ENV SRC_EXTENSIONS_DIR=/srv/app/src_extensions
 
 # Install packages needed by the dev requirements
-RUN apk add --no-cache libffi-dev netcat-openbsd
+RUN apk add --no-cache libffi-dev
 
 # Install CKAN dev requirements
 RUN echo $GIT_BRANCH
 RUN pip install --no-binary :all: -r https://raw.githubusercontent.com/ckan/ckan/${GIT_BRANCH}/dev-requirements.txt
 
+# cryptography 2.9 is broken with this error message
+# ImportError: Error relocating /usr/lib/python2.7/site-packages/cryptography/hazmat/bindings/_openssl.so: RSA_get0_crt_params: symbol not found
```

Comment:
This suggests that `cryptography` 2.9 is designed for use with openssl 1.1.x. The base alpine image must have 1.0.x

Reply:
Probably, for now it's ok to use that as it will require a bit of work to put in place.

Reply:
FWIW (and you should know this was coming) Nix solves these sorts of issues with basically zero effort.